backend/infrahub/core/diff/coordinator.py

```diff
@@ -217,12 +269,10 @@ async def _update_diffs(
             if diff_uuids_to_delete:
                 await self.diff_repo.delete_diff_roots(diff_root_uuids=diff_uuids_to_delete)
 
-        for enriched_diff in aggregated_diffs_by_branch_name.values():
```

Comment:
moved this logic to add summaries and save a diff out of this method b/c the new `recalculate` method needs to transfer conflicts to the new version of the diff before adding summaries and saving